app/src/test/java/swati4star/createpdf/util/ImageUtilsTest.java

```diff
@@ -0,0 +1,60 @@
+package swati4star.createpdf.util;
+
+import com.itextpdf.text.Rectangle;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class ImageUtilsTest {
+    private static final Rectangle NO_NORMAL_ERROR =
+            new Rectangle(0.0f, 0.0f, 0.0f, 0.0f);
+    private static final Rectangle NO_POINT_ERROR =
+            new Rectangle(0.0f, 0.0f, 0.0f, 0.0f);
+    private static final Rectangle NO_LINE_ERROR =
+            new Rectangle(0.0f, 0.0f, 0.0f, 0.0f);
```

Comment:
I am not able to understand what these three rectangle's mean especially since they have everything equal to 0.0f. Can you explain please?

Reply:
They are just to signify different errors to different comparisions. It can be done with a single `NO ERROR` variable too.